### ISSUE TITLE
DHF 5.0.0 - when executing mlDeployDatabases, the datahubCreated* fields are removed

### ISSUE THREAD

**Issue by derms** [marklogic/marklogic-data-hub#2530]
#### The issue

when executing mlDeployDatabases that has fields configured, the datahubCreated* fields are removed

Sample indexes defined in src/main/ml-config/databases/final-database.json

```
{
   "database-name":"%%mlFinalDbName%%",
   "range-element-index":[

   ],
   "schema-database":"%%mlFinalSchemasDbName%%",
   "triggers-database":"%%mlFinalTriggersDbName%%",
   "triple-index":true,
   "collection-lexicon":true,
   "uri-lexicon":true,
   "range-field-index":[
      {
         "scalar-type":"dateTime",
         "field-name":"systemStart",
         "collation":"",
         "range-value-positions":true,
         "invalid-values":"reject"
      },
      {
         "scalar-type":"dateTime",
         "field-name":"systemEnd",
         "collation":"",
         "range-value-positions":true,
         "invalid-values":"reject"
      }
   ],
   "field":[
      {
         "field-name":"systemStart",
         "metadata":"true",
         "word-lexicons":null,
         "included-elements":null,
         "excluded-elements":null,
         "tokenizer-overrides":null
      },
      {
         "field-name":"systemEnd",
         "metadata":"true",
         "word-lexicons":null,
         "included-elements":null,
         "excluded-elements":null,
         "tokenizer-overrides":null
      }
   ]
}
```

Before mlDeployDatabases

![image](https://user-images.githubusercontent.com/1885934/58164639-7e6c4f00-7c86-11e9-81c5-086463f949f5.png)

After mlDeployDatabases

![image](https://user-images.githubusercontent.com/1885934/58164672-8c21d480-7c86-11e9-9382-940ecd237e6c.png)

#### Tech details

Which Operating System are you using? Mac

Which version of MarkLogic are you using? 9.0.9-1

Which version of the Data Hub Framework are you using? 5.0.0

Output below
[myoutput.txt](https://github.com/marklogic/marklogic-data-hub/files/3207114/myoutput.txt)

**Comment by rjrudin** [comment]
@derms This is the same issue as DHFPROD-2554

**Comment by derms** [comment]
@rjrudin ok great thanks for letting me know

**Comment by aebadirad** [comment]
Fixed in 5.0.1
